config/recipes/beats/filebeat_autodiscover.yaml

```diff
@@ -75,6 +75,20 @@ rules:
   - get
   - watch
   - list
+- apiGroups: ["apps"]
+  resources:
+  - replicasets
+  verbs:
+  - get
```

Comment:
Nit: in https://github.com/elastic/cloud-on-k8s/pull/6958, I didn't give permission to `get`. It we need `get` here, it would mean that this is slightly different. I'm fine with giving that permission, but the less the better in terms of permission.

Reply:
I used the k8s manifest from the Beats team as a reference.